lib/date_time/date_time.c

```diff
@@ -105,7 +120,7 @@ static int time_modem_get(void)
 
 	/* Want to match 6 args */
 	if (rc != 6) {
-		LOG_ERR("Could not get cellular network time, error: %d", rc);
+		LOG_DBG("Could not get cellular network time, error: %d", rc);
```

Comment:
@simensrostad I changed this back to LOG_DBG from LOG_ERR. It had been changed in commit https://github.com/nrfconnect/sdk-nrf/commit/6513b764ffe534eeb31d9f670d91f33706a7e840
Change was made to align with our discussion that modem time retrieval failures shouldn't raise too much attention as they are normal in many networks.

We still have somewhat duplicate logging here and in the caller but I didn't change that:
```
<dbg> date_time.time_modem_get: Could not get cellular network time, error: -77
<dbg> date_time.new_date_time_get: Did not get time from cellular network. This is normal as some cellular networks don't provide it.
```

Reply:
I'll leave the log output change up to you.

Reply:
I've done a bit more changes to logging output as we discussed.